demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -790,6 +845,7 @@ export class DemoMeetingApp implements
   }
 
   leave(): void {
+    this.statsCollector.resetStats();
```

Comment:
Does leave get called if someone else ends the meeting? I see we only call leave() on the meetingEnd and meetingLeave button

Reply:
Yeah it does not, have added the `this.statsCollector.resetStats();` to `audioVideoDidStop` as well. Thanks for catching it!